Changed from field injection to constructor injection

sabaubogdan opened the pull request.

sabaubogdan committed:
Changed from field injection to constructor injection

DavidArt commented:
looks ok. you can do the merge

sabaudragos commented on src/main/java/xyz/vegaone/easytracking/config/EasyTrackConfig.java:
```diff
@@ -23,5 +23,4 @@ public DataSource dataSource() {
 
```
For some reason I cannot comment on line 14th, but you have a field injection there. Could you please switch to constructor based injection?

sabaudragos commented on src/main/java/xyz/vegaone/easytracking/service/ProjectService.java:
```diff
@@ -19,21 +19,26 @@
 @Service
 public class ProjectService {
 
```
There are some unused imports in this class and some commented code at line 53, could you please remove them?

sabaudragos commented on src/test/java/xyz/vegaone/easytracking/project/ProjectServiceTest.java:
```diff
@@ -28,12 +28,16 @@
+        this.projectService = projectService;
+        this.userStoryService = userStoryService;
+    }
+
```
This class has some unused imports and commented code as well. Could you please remove them?

sabaubogdan committed:
Removed commented code and unused code/imports
Changed to field injection in unit tests (change from constructor injection to field injection)
Changed to constructor injection in EasyTrackConfig
